.eleventy.js

```diff
@@ -1,9 +1,11 @@
-const fs = require("fs");
+const fs = require("node:fs");
+const path = require("node:path");
 
 module.exports = (eleventyConfig) => {
-  eleventyConfig.addShortcode("photoFolder", function (photoFolder) {
-    const html = fs.readdirSync(photoFolder)
-      .map((file) => `<img src="${file}" />`);
+  eleventyConfig.addShortcode("photoFolder", function (photoFolder, extensions = [".png", ".gif"]) {
+    const html = fs.readdirSync(photoFolder, {withFileTypes: true})
+      .filter(file => file.isFile() && extensions.includes(path.extname(file.name)))
```

Comment:
This will ignore non-files (ie: subfolders) and will only return the specified file extensions from `extensions[]` argument.